shared/src/testFixtures/groovy/gov/hhs/cdc/trustedintermediary/context/TestApplicationContext.groovy

```diff
@@ -19,8 +19,9 @@ class TestApplicationContext extends ApplicationContext {
         TEST_ENV_VARS.clear()
     }
 
-    def static injectRegisteredImplementations() {
-        injectRegisteredImplementations(true)
+    def static injectRegisteredImplementations(def skip = true) {
```

Comment:
We'd never need to set `skip` to `false` in a testing context.  I think we can remove the argument from this method.

The skipping allows us to only need to register implementations that are in use in each specific test.  If we don't allow skipping, each test needs to register every `@Inject` field across the entire application.  For example, this means that a test would need to register an `HttpClient` implementation even if the test isn't anywhere close to the HTTP code.